### ISSUE TITLE
fmovie is installed, but impossible to launch

### ISSUE THREAD

**Issue by MathieuMauron** [coder-alpha/FMoviesPlus.bundle#72]
is there any settings to set to launch fmovie ?

![Sans titre](https://user-images.githubusercontent.com/42377268/63594079-c9fc0500-c5b5-11e9-82c0-d0719f1caf24.png)

possible to clic on cog and cross only

**Comment by Oldpapa** [comment]
New server probably has disabled plugins which has been promised by plex. Plugins are being removed. Go back on server

Sent from Yahoo Mail on Android 
 
  On Fri, Aug 23, 2019 at 6:53 AM, MathieuMauron<notifications@github.com> wrote:   
is there any settings to set to launch fmovie ?



possible to clic on cog and cross only

**Comment by pdille** [comment]
Yeah, I can confirm this. It seems to have cropped up in 1.16.5.1554

Downgrade to [1.16.5.1488](https://downloads.plex.tv/plex-media-server-new/1.16.5.1488-deeb86e7f/windows/PlexMediaServer-1.16.5.1488-deeb86e7f-x86.exe) (or older)
